### PR TITLE
create scala wrapper for digdag config

### PR DESCRIPTION
**Pull request by mocyuto**
create Scala wrapper for Digdag config for remove guava Optional

### CODE REVIEW

File: src/main/scala/pro/civitaspo/digdag/plugin/ecs_task/register/EcsTaskRegisterOperator.scala [hunk 1]
```diff
@@ -1,409 +1,222 @@
...
-  SystemControl,
-  Tag,
-  TaskDefinitionPlacementConstraint,
-  Tmpfs,
-  Ulimit,
-  Volume,
-  VolumeFrom
-}
-import com.google.common.base.Optional
+import com.amazonaws.services.ecs.model._
```
Comment (civitaspo) [inline src/main/scala/pro/civitaspo/digdag/plugin/ecs_task/register/EcsTaskRegisterOperator.scala:2]: Avoid wildcard imports

File: src/main/scala/pro/civitaspo/digdag/plugin/ecs_task/register/EcsTaskRegisterOperator.scala [hunk 2]
```diff
@@ -1,409 +1,222 @@
...
-    if (ipcMode.isPresent) throw new UnsupportedOperationException("Currently aws-java-sdk does not support ipc_mode.")
-    if (memory.isPresent) req.setMemory(memory.get)
-    if (networkMode.isPresent) req.setNetworkMode(networkMode.get)
-    if (pidMode.isPresent) throw new UnsupportedOperationException("Currently aws-java-sdk does not support pid_mode.")
-    if (placementConstraints.nonEmpty) req.setPlacementConstraints(placementConstraints.asJava)
-    if (requiresCompatibilities.nonEmpty) req.setRequiresCompatibilities(requiresCompatibilities.asJava)
-    if (tags.nonEmpty) req.setTags(tags.asJava)
-    if (taskRoleArn.isPresent) req.setTaskRoleArn(taskRoleArn.get)
-    if (volumes.nonEmpty) req.setVolumes(volumes.asJava)
+    req.setFamily(c.get("family"))
```
Comment (civitaspo) [inline src/main/scala/pro/civitaspo/digdag/plugin/ecs_task/register/EcsTaskRegisterOperator.scala:19]: could you follow the document order? See https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_RegisterTaskDefinition.html .

**Review by civitaspo — CHANGES_REQUESTED** [review]
Good catch! I agree your implementation, so please apply other classes.

**Comment by civitaspo** [comment]
How's this going? @moc-yuto

**Comment by mocyuto** [comment]
mmm, too many to fix...
I close this PR...
If you agree it, please implement like this! 🙇

**Pull request closed without merging**